homeassistant/components/kulersky/light.py

```diff
@@ -158,6 +158,11 @@ async def async_update(self) -> None:
         if not brightness:
             self._attr_rgbw_color = (0, 0, 0, 0)
         else:
-            rgbw_normalized = tuple(round(x * 255 / brightness) for x in rgbw)
-            self._attr_rgbw_color = rgbw_normalized  # type:ignore[assignment]
+            rgbw_normalized = [round(x * 255 / brightness) for x in rgbw]
+            self._attr_rgbw_color = (
```

Comment:
Did you try casting `rgbw` to a four item tuple?

Reply:
Yes I did, but the `normalisation`/`rounding` routine loses the information and results in `list[int]` or `tuple[int, ...]`.

Reply:
Ok.